page/ajax/working-with-jsonp.md

```diff
@@ -31,4 +31,8 @@ $.ajax({
 });
 ```
 
+If you'd like to access community tables you have to add `env: "http://datatables.org/alltables.env"` to the data object otherwise you would get an error message with `No definition found for Table`.
```

Comment:
This isn't the first time we're in need of a good, fake API to demonstrate Ajax to people (https://github.com/jquery/learn.jquery.com/issues/475 for example). I'm going to drop it on the meeting notes so the team can brain storm about it.

I just feel like this might already be too complicated for some people, plus, people are here to learn about jQuery - not about the way YQL works.